domain-models-runtime/src/test/java/org/folio/rest/persist/PgUtilIT.java

```diff
@@ -1107,6 +1107,49 @@ public void optimizedSQLwithNo500(TestContext testContext) {
         }));
   }
 
+   @Test
+  public void deleteByCQLwithNo500(TestContext testContext) {
+    PostgresClient pg = PostgresClient.getInstance(vertx, "testtenant");
+    insert(testContext, pg, "delete_test",  1);
+    PgUtil.delete("users",  "username=delete_test",
+        okapiHeaders, vertx.getOrCreateContext(), ResponseWithout500.class, testContext.asyncAssertFailure(e -> {
+          assertThat(e, is(instanceOf(NullPointerException.class)));
```

Comment:
The assert should check that the message contains an indication the that respond500WithTextPlain method is missing. See deleteByIdResponseWithout500 for an example.

Reply:
done